src/setup/config.jl

```diff
@@ -7,13 +7,26 @@ function __init__()
     if !isdir(odinn_path)
         mkpath(odinn_path)
     end
-    if !isdir(prepro_dir)
+    existsAndRedownload = false
+    if isdir(prepro_dir)
+        daysSinceLastDownload = (Dates.now() - Dates.unix2datetime(mtime(prepro_dir)))/Day(1)
+        if daysSinceLastDownload > 1
+            # Re-download data if older than one day
```

Comment:
Isn't this a bit extreme? Why should files change that much after just one day?

Reply:
OK, I think this is now doable because we just have few data in the google drive. If we implemented this with the Bremen server would it attempt to download the whole thing? Or just the necessary `gdir`s?